ogr/services/gitlab.py

```diff
@@ -26,6 +26,11 @@ def __init__(self, token=None, url=None, full_repo_name=None):
         if full_repo_name:
             self.repo = self.g.projects.get(full_repo_name)
 
+    def __str__(self) -> str:
+        return(
+            f"GitLab Service"
```

Comment:
The GitLab currently does not work. We can work on the `__str__` after it is ready.

Reply:
Should I leave as it is for now ? or should I remove the method ?

Reply:
You can remove it. The attributes may change before we make this instance work...

Reply:
Yes, I will do it.